usaspending_api/download/management/commands/populate_monthly_delta_files.py

```diff
@@ -284,12 +302,29 @@ def add_arguments(self, parser):
                                            'Defaults to both.')
         parser.add_argument('--last_date', dest='last_date', default=None, type=str, required=True,
                             help='Date of last Delta file creation.')
+        parser.add_argument('--remove_transaction_deltas',
+                            action='store_true',
+                            help='Empties out transactions from the transaction_deltas file in the '
+                            'database indicated by the TRANSACTION_DELTA_URL environment variable. '
+                            'This will typically be a staging or production database.')
 
     def handle(self, *args, **options):
         """ Run the application. """
         agencies = options['agencies']
         award_types = options['award_types']
         last_date = options['last_date']
+        remove_transaction_deltas = options['remove_transaction_deltas']
+        transaction_delta_url = os.environ.get('TRANSACTION_DELTA_URL')
```

Comment:
There have been a movement away from individual scripts directly accessing env vars. All configs should either be provided by cli args or settings.py